src/zod/index.ts

```diff
@@ -2,6 +2,7 @@ export {
   Abi,
   AbiConstructor,
   AbiEvent,
+  AbiEventParameter,
```

Comment:
Let's add to the docs too: https://abitype.dev/api/zod.html#schemas

Reply:
Sorry my bad. Just added it.